output/quay.go

```diff
@@ -47,6 +48,8 @@ type Vuln struct {
 	Description   string    `json:"Description"`
 	Name          string    `json:"Name"`
 	Metadata      *Metadata `json:"Metadata"`
+	// This is a departure from the "spec" but needed for Konflux
```

Comment:
what would it be if it were following the spec?

Reply:
There is no official spec but the current Quay secscan response does not include an `Issued` at all.